docs/docusaurus.config.js

```diff
@@ -43,7 +43,7 @@ const config = {
       /** @type {import('@docusaurus/preset-classic').Options} */
       ({
         docs: {
-          path: "processed-docs",
+          path: process.env.ENV === "dev" ? "docs" : "processed-docs",
```

Comment:
Sure, in that case you don't set that env 👍 (ex by running `yarn start:dev:local` instead of `yarn start:dev`). We don't get hot reloads if we don't look at the docs folder.